src/app/services/baw-api/audio-recording/audio-recordings.service.ts

```diff
@@ -90,6 +93,25 @@ export class AudioRecordingsService extends ReadonlyApi<AudioRecording> {
       ) as Filters
     );
   }
+
+  public downloadUrl(audioRecording: IdOr<AudioRecording>) {
+    return this.getPath(audioRecordingOriginalEndpoint(audioRecording));
+  }
+
+  /**
+   * Returns a link to the API which will download a templated script for downloading original audio files
+   *
+   * @param filter Audio recording filters
+   */
+  public batchDownloadUrl(filter: Filters<AudioRecording>): string {
+    const qsp = new URLSearchParams({
+      // Base64 RFC 4648 §4 encoding
```

Comment:
```suggestion
      // Base64 RFC 4648 §5 encoding
```

Reply:
This suggestion was correct, base64url is section 5 of the RFC. Why did you reject it? https://datatracker.ietf.org/doc/html/rfc4648#section-5